client/package.json

```diff
@@ -46,6 +46,8 @@
     ]
   },
   "devDependencies": {
+    "@babel/core": "^7.12.9",
+    "@babel/preset-env": "^7.12.7",
```

Comment:
What are we using Babel for?

Reply:
oops, I didnt add that manually. I am going to delete it then.

Reply:
Cool, everything else looks good.